modules/core/core-scheduler/src/main/resources/com/enonic/xp/impl/scheduler/default.properties

```diff
@@ -0,0 +1,5 @@
+job.audit.enabled=true
+job.audit.calendar.type=cron
+job.audit.calendar.value=0 1 * * *
+job.audit.calendar.timezone=GMT+2:00
```

Comment:
It would be daytime in some areas. We may need to find a way to specify "local time"